docroot/themes/contrib/civic/civic-library/components/01-atoms/button/button.twig

```diff
@@ -30,12 +31,14 @@
 {% set size = size in ['extra-large', 'large', 'regular', 'small', 'extra-small'] ? size : 'regular' %}
 {% set icon_placement = icon_placement in ['before', 'after'] ? icon_placement : 'after' %}
 {% set icon_size = icon_size in ['extra-large', 'large', 'regular', 'small', 'extra-small'] ? icon_size : size %}
-
+{% set component_name = type == 'chip' ? 'filter-chip' : 'button' %}
```

Comment:
This might seem weird but filter chips are not always radio or checkbox (look at large-filter.js) because for views forms we don't want the selected filters showing up in the URL.

So we need filter chip buttons as well as checkboxes - I don't like this

Reply:
firstly - please do not use `filter-chip` code in any code. this is an "invention" by designers - it should be never ever mentioned anywhere. The button type is just `chip`.

secondly - "chips are not always radio or checkbox" - feels like there is a miss-understanding on what FE components are: they have nothing to do with what and how is shown in the URL - that problem needs to be solved outside of the context of the button component.